pcsx2/Sio.cpp

```diff
@@ -31,10 +31,6 @@ static void sioWrite8inl(u8 data);
 // Magic psx values from nocash info
 static const u8 memcard_psx[] = {0x5A, 0x5D, 0x5C, 0x5D, 0x04, 0x00, 0x00, 0x80};
 
-// Memory Card Specs for standard Sony 8mb carts:
-//    Flags (magic sio '+' thingie!), Sector size, eraseBlockSize (in pages), card size (in pages), xor checksum (superblock?), terminator (unused?).
-static const mc_command_0x26_tag mc_sizeinfo_8mb= {'+', 512, 16, 0x4000, 0x52, 0x5A};
```

Comment:
One thing about removing this one is that it's documenting specs I could potentially see someone working on the code using...

Reply:
What about just commenting out the code then ?

Reply:
Commenting it out is probably fine. I've left commented code in plenty of times when it looks useful for reference or logging/debugging.